roles/nova-data/tasks/libvirt.yml

```diff
@@ -56,29 +58,157 @@
   notify: restart nova services
   when: openstack_install_method == 'source'
 
+- name: make pki paths
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: kvm
+  with_items:
+    - /etc/pki/CA
+    - /etc/pki/libvirt/private
+
+- name: get uname -n
+  command: uname -n
+  register: uname
+
+- name: create 'uname -n' as fact 'uname_host'
+  set_fact:
+    uname_host: "{{ uname.stdout }}"
+
+- name: install CA file
+  template:
+    src: cacert.pem.j2
+    dest: /etc/pki/CA/cacert.pem
+    owner: root
+    group: kvm
+    mode: 0644
+  register: ca_install
+
+- name: upload CA key
+  template:
+    src: cacert.key.j2
+    dest: /tmp/cacert.key
+  register: ca_key_install
+
+- name: upload server certificate template
+  template:
+    src: server-cert-template.j2
+    dest: /tmp/server-cert-template
```

Comment:
We delete them anyway.  I think it's good to delete them rather than leave them around - I don't think they have any value in `/etc/`, and I figure `/tmp` is the best place for ephemeral stuff.